### ISSUE TITLE
Problem with Quantum manager on site Joomla 4.1.5

### ISSUE THREAD

**Issue by Olivier834** [Quantum-Manager/tracker#128]
**Describe the bug**
When I open an image folder from Quantum manager, the images do not appear above their name. Whereas when I open the same images folder in administration/media/images the thumbnail images appear way above the name.

**To Reproduce**
1. Go to 'administration/components/Quantum manager'

**Expected behavior**
I would like all images in the folder to appear

**Screenshots**
![Quantum-manager](https://user-images.githubusercontent.com/65899195/185980872-3988fc07-b9b9-4f6c-acd3-d78aa28ae6d4.jpg)

**System Information :**
 - Extensions version Quantum manager 2.0.0
 - PHP version: 8.0 or 7.4
 - Server OS: windows 11
 - Browser Firefox or Chrome or Edge, or Safari on iPad

**Additional context**
access provider Ionos, Site Joomla 4.1.5

**Comment by dmitriitux** [comment]
you have akeeba tools on site?

**Comment by Olivier834** [comment]
yes, I have admin tools from Akeeba but I don't really know how to use it...

**Comment by dmitriitux** [comment]
this tool blocks cache folder in .htaccess rules, thumbnail previews are saved in joomla cache

**Comment by Olivier834** [comment]
I have two sites hosted at Ionos. The second is a copy of the first but on the second I installed admin tools and not on the first.
However, on both sites I have this Quantum manager bug...

**Comment by Sulpher** [comment]
To check out the theory that the issue is caused by htaccess generated by Admin Tools,  please try to rename .htaccess to any other temporary name like .htaccess2.  (the SEF will no longer working on the front-end, it is temporary)
And go to the administrator panel and check out on possible issues with thumbs. If all is fine and there is no longer such an issue, then we found the reason - it is .htaccess and you need to use default Joomla htaccess file or go this way to investigate which command causes the issue.
If renaming .htaccess will not help, then there is another issue.

**Comment by Sulpher** [comment]
Btw, do you have issues with thumbnails everywhere in all folders or only in Paysage5 folder?

**Comment by Olivier834** [comment]
Thanks for these tips. I will investigate on Sunday.
My .htaccess file is that of "aesecure".
On a brand new and empty Joomla 4.2.0 site I installed Quantum manager and the thumbnails are displayed correctly.
Moreover, I note that on a site installed under Wamp, Quantum manager does not work; it says "Loading" and spins indefinitely...

**Comment by Sulpher** [comment]
You are welcome.
Admin Tools changes .htaccess files by adding extra commands, but as far as I remember, there are some folders that might be excluded, so you need a correct set up of Admin Tools. Alternatively, you can use default Joomla .htaccess file.

Well, as for the 2nd question: 
1. Try to see errors in browser console.
2. Try to see error.log of local server (not access.log).

**Comment by dmitriitux** [comment]
@Olivier834 

what is Wamp?

**Comment by Olivier834** [comment]
Thanks for these tips. I will investigate on Sunday.
My .htaccess file is that of "aesecure".
On a brand new and empty Joomla 4.2.0 site I installed Quantum manager and the thumbnails are displayed correctly.
Moreover, I note that on a site installed under Wamp, Quantum manager does not work; it says "Loading" and spins indefinitely...

**Comment by Olivier834** [comment]
[WampServer](https://www.wampserver.com/en/download-wampserver-64bits/) is a Windows web development environment. It allows you to create web applications with Apache2, PHP and a MySQL database. Alongside, PhpMyAdmin allows you to manage easily your databases

**Comment by dmitriitux** [comment]
@Olivier834 hmm, maybe there are not enough modules for php or some special configuration. Can you somehow give access to look at such a server? Or is he only on local computer? I just have all the development on macos, there is no Windows configured anywhere, only if I raise the virtual machine

**Comment by Sulpher** [comment]
On French forum there is an example:
`/administrator/cache/com_quantummanager/images/stories/Bug Quantum manager/madeleine.jpg`

So, it is curious why there is **/Bug Quantum manager/** folder with upper case letters and space symbol in /cache/ folder... 
I suppose this is the reason why thumbnails aren't displayed, but we need to understand why such a folder was generated.

**Comment by Olivier834** [comment]
1.- Thank you for your contributions.
I had two sites hosted at Ionos. The first was created a long time ago in Joomla 1.5. It was in 4.1.5 and Quantum manager did not display thumbnails. The second was created directly in Joomla 4.2.0 then I installed the same extensions as on the first and I imported the database from the first. Quantum manager works perfectly there. So I overwritten the first site with the second and everything works perfectly.
I don't know the cause of the bug but I imagine it's a remnant of Joomla extension which was incompatible with QM.
2.- As for Wamp, Wamp allows you to build and test sites locally on your PC. I cannot therefore give you access to the sites that are hosted there.
I confirm that when you want to open the Quantum manager component, it displays "Loading" and runs indefinitely. For me personally, this is not a problem because QM works on the internet. It was just a comment to you.
3.- The Bug Quantum manager folder, whose name is written with a capital letter and spaces, now opens without any problem in QM, despite capital letters and spaces...
